.github/workflows/cross-compiles.yml

```diff
@@ -132,7 +138,7 @@ jobs:
             tests: none
           }
         ]
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
This changes things for all the archs.  Do we want that?

Reply:
currently `ubuntu-latest` is and has always been `ubuntu-20.04`
but the change to `ubunto-22.04` is imminent, and it will break a lot
of things, not only the sparc64-linux-gnu qemu target but also other
targets as well, e.g. m68k-linux-gnu would break due to a false-positive
--strict-warnings issue in gcc11.
So yes sure, we will want that.

Reply:
We could always include the OS as part of the matrix.  That's probably a good idea because it means we can test older versions in addition to newer ones.

Reply:
Another PR is okay.

Reply:
Please drop this change.

Reply:
Then add another field for the matrix parameters with ubuntu-latest as default?

Reply:
Yes please.